style/about.css

```diff
@@ -0,0 +1,490 @@
+    font-style: normal;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 14px;
+    display: flex;
+    align-items: center;
+    color: #fff;
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+  }
+
+  .campagin {
+    display: flex;
+    align-items: center;
+    color: #ec4327;
+    border: 4px solid #ec4327;
+    width: 125px;
+    padding: 4px 14px;
+    white-space: nowrap;
+  }
+
+  .hero-logo {
+    padding-left: 220px;
+    padding-top: 92px;
+  }
+
+  .navbar {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 0 198px 0 209px;
+    gap: 232px;
+    width: 100%;
+    height: 75px;
+    background: #fff;
+  }
+
+  /* Navbar banner */
+  .nav-banner-item a {
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .nav-banner-item a:hover {
+    text-decoration: underline;
+  }
+
+  .nav-item-icon {
+    gap: 20px;
+  }
+
+  .logo {
+    height: 51px;
+    width: 130px;
+    left: 205px;
+    top: 12px;
+    border-radius: 0;
+    display: block;
+  }
+
+  .hero-container {
+    width: 100%;
+    height: 887px;
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    padding: 0 3%;
+    background-image: url(../images/mobile-background.png);
+  }
+
+  .hero-container-about {
+    width: 100%;
+    min-height: 40vh;
+    background-repeat: no-repeat;
+    background-size: 100%;
+    padding: 0 3%;
+    background-image: url(../images/about_bg_main.png);
+  }
+
+  .auto-layout {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: auto;
+    background-color: #f7f7f8;
+  }
+
+  .mobile-logo {
+    display: none;
+  }
+
+  .desktop-logo {
+    display: block;
+    margin-right: 200px;
+    margin-left: 11px;
+  }
+
+  .sub-team {
+    box-sizing: border-box;
+    width: 923px !important;
+    height: 200px !important;
+    background: #fff;
+    border: 8.49673px solid #fff;
+    margin-top: 69px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    margin-left: 100px;
+    margin-right: 258px;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 35px 65px;
+    gap: 10px;
+  }
+
+  .event-date {
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    height: 18px;
+    margin-top: 25px;
+    letter-spacing: 0.03em;
+    padding-left: 200px;
+    width: 1440px;
+  }
+
+  .event-discription {
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 14px;
+    line-height: 25px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.02em;
+    text-decoration-line: underline;
+    color: #5f5f5f;
+    width: 229px;
+    height: 14px;
+    padding-left: 400px;
+    margin-top: 15px;
+    padding-bottom: 70px;
+  }
+
+  .section-heading {
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
```

Comment:
- [Optional] COnsider using classes to avoid repeating yourself. I notice that you repeatedly used these two properties in your document. Consider creating a class for it and use the class for your elements.